src/client/v2/indexer/lookupAccountTransactions.ts

```diff
@@ -119,7 +119,7 @@ export default class LookupAccountTransactions extends JSONRequest {
    *        .txid(txId)
    *        .do();
    * ```
-   *
+   * @remarks Alternatively, use `indexerClient.lookupTransactionByID(txnId).do()`
```

Comment:
A few queries seemed to perform the same request as another more straightforward method so I added remarks to them